source/ast/expressions/AssertionExpr.cpp

```diff
@@ -1178,6 +1539,14 @@ void CaseAssertionExpr::serializeTo(ASTSerializer& serializer) const {
         serializer.write("defaultCase", *defaultCase);
 }
 
+/// Returns `nullopt` if `disable iff` condition is always `true` (`1`)
+std::optional<SequenceRange> DisableIffAssertionExpr::computeSequenceLengthImpl() const {
+    if (condition.constant && condition.constant->integer().countOnes())
```

Comment:
The `ConstantValue::isTrue()` function is probably a nicer way to express this than calling countOnes()

Reply:
fixed